Readme.md

```diff
@@ -43,30 +44,18 @@ Comparison
 
 A small echo server, parses and returns the url query parameters:
 
-- [restiq](https://www.npmjs.org/package/restiq) - 20.9k/s
-- [http](https://nodejs.org/api/http.html) - 17.6k/s
-- [express](https://www.npmjs.org/package/express) - 7.9k/s
-- [restify](https://www.npmjs.org/package/restify) - 4.6k/s (8k/s using just the http methods)
-- [hapi](https://www.npmjs.org/package/hapi) - 0.2k/s* (1.8k/s with `setNoDelay()`)
+- restiq - 20.9k/s
```

Comment:
the npmjs page has a link to github, but github doesn't link to npmjs, so I tend to
point to npmjs (that, and that's the version most people would install)